### PR TITLE
Low Impact Flash-Based CSRF updates

### PR DESCRIPTION
**Pull request by RRudder**
update to template.md with breakout recommendations

### CODE REVIEW

File: submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/low_impact/template.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Flash-Based Cross-Site Request Forgery (Low Impact)
+
+## Overview of the Vulnerability
+
+Cross-Site Request Forgery (CSRF) occurs when requests to the application are submitted on behalf of an authenticated user without their knowledge via crafted, malicious code which can be in the form of a link the user clicks. The application is unable to distinguish between the malicious request and a legitimate request from the user.
+
+A low impact flash-based CSRF is possible for this application, allowing an attacker to submit requests to the application on behalf of an authenticated user. Additionally, the attacker needs to socially engineer the user to click on a link, or paste the malicious code into the user’s browser. If successful, the code will execute within that user’s browser in the context of this domain.
```
Comment (evildaemond) [inline submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/low_impact/template.md:7]: This seems to be lacking what make sit High Impact vs Low Impact, as we need to specify what the difference is between them for the client and researcher.

File: submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/low_impact/template.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## Steps to Reproduce
+
+1. Enable a HTTP interception proxy, such as Burp Suite or OWASP ZAP
+1. Use a browser to sign into the application at: {{URL}}
+1. Navigate to the following URL and submit the form: {{URL}}
+1. Use the HTTP interception proxy to intercept the request triggered by the form
+1. Observe that a request is sent using `.SWF` file:
+
+{{screenshot}}
+
```
Comment (evildaemond) [inline submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/low_impact/template.md:22]: This seems redundant for triage, at the end of the day, we only require the SWF file, and sometimes a link, not the creation.

**Review by evildaemond — CHANGES_REQUESTED** [review]
2 comments then LGTM

File: submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/low_impact/template.md (hunk 2 above)
Comment (RRudder) [reply submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/low_impact/template.md]: addressed, ty

File: submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/low_impact/template.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/low_impact/template.md]: Let me know if the latest commit satisfies the details around low impact

**Review by evildaemond — APPROVED** [review]
LGTM

**Pull request merged**